devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy

```diff
@@ -62,6 +62,11 @@ metadata {
 		fingerprint mfr: "0312", prod: "C000", model: "CO05", deviceJoinName: "Evalogik Outlet", ocfDeviceType: "oic.d.smartplug" //Evalogik Mini Outdoor Smart Plug
 		fingerprint mfr: "031E", prod: "0004", model: "0001", deviceJoinName: "Inovelli Switch" //Inovelli Switch
 		fingerprint mfr: "001D", prod: "0037", model: "0002", deviceJoinName: "Leviton Outlet", ocfDeviceType: "oic.d.smartplug" //Leviton Tamper Resistant Outlet ZW15R
+        fingerprint mfr: "0312", prod: "FF00", model: "FF0C", deviceJoinName: "Minoston Mini Plug", ocfDeviceType: "oic.d.smartplug" //Minoston Mini Smart Plug
+        fingerprint mfr: "0312", prod: "EE00", model: "EE01", deviceJoinName: "Minoston Smart Switch", ocfDeviceType: "oic.d.smartplug" //Minoston Smart Switch
+        fingerprint mfr: "0312", prod: "EE00", model: "EE03", deviceJoinName: "Minoston Smart Switch", ocfDeviceType: "oic.d.smartplug" //Minoston Smart on/off Toggle Switch
```

Comment:
**Applies to all new fingerprints:** 
Please add a Raw Description in a comment after the fingerprint so it would look like, for example:
fingerprint mfr: "0312", prod: "E", model: "0900", deviceJoinName "Minoston Switch" // Raw Description (paste the raw description here)

(Raw description can be found at the Smartthings graphapi in device details)

Reply:
@sky-nie any updates here ?